proto/gz/msgs/imu.proto

```diff
@@ -37,4 +38,7 @@ message IMU
   Quaternion orientation       = 3;
   Vector3d angular_velocity    = 4;
   Vector3d linear_acceleration = 5;
+  Float_V orientation_covariance         = 6;
```

Comment:
Since the respective values are stored in `double`, it probably makes sense for the corresponding covariance to be stored in `double`.

these are also going to need documentation in terms of conventions/ordering here, it probably makes sense to stick close the ROS coordinate conventions described here: https://github.com/ros2/common_interfaces/blob/rolling/sensor_msgs/msg/Imu.msg

Finally, a note that just adding the fields won't be enough to get them populated, that support will also have to be added in gz-sensors.

Reply:
Thanks for the quick review.
I'll change the order asap.
However, I didn't find any example of a message with double array, so I used the same type as the one used in other messages that contain covariance: [PoseWithCovariance](https://github.com/gazebosim/gz-msgs/blob/gz-msgs9/proto/gz/msgs/pose_with_covariance.proto#L39) and [TwistWithCovariance](https://github.com/gazebosim/gz-msgs/blob/gz-msgs9/proto/gz/msgs/twist_with_covariance.proto#L39)
Also, as mentioned, I just created 2 other PR that will populate those messages.